.github/workflows/cspell.yml

```
@@ -0,0 +1,24 @@
+name: CSpell
+on: push
+jobs:
+  cspell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
```

Cache support is [built in](https://github.com/actions/setup-node/blob/25316bbc1f10ac9d8798711f44914b1cf3c4e954/docs/advanced-usage.md#caching-packages-dependencies) to `actions/setup-node@v2`
```suggestion
          node-version: '14'
          cache: 'yarn'
          cache-dependency-path: yarn.lock
```

---

Done!